client/packages/common/src/ui/components/portals/AppFooter/AppFooter.tsx

```diff
@@ -26,8 +27,10 @@ export const AppFooter: FC = () => {
     setAppSessionDetailsRef(appSessionDetailsRef);
   }, []);
 
+  const hideFooter = fullScreen || keyboardOpen;
+
   return (
-    <Box sx={{ display: fullScreen ? 'none' : undefined }}>
+    <Box sx={{ display: hideFooter ? 'none' : undefined }}>
```

Comment:
When not in modal, i.e. typing in a table filter, hide the footers so can see more of the results